rotator.cpp

```diff
@@ -80,6 +89,11 @@ Rotator *Rotator::Factory(const wxString& choice)
         else if (choice.Find(_T("ASCOM")) != wxNOT_FOUND) {
             rotator = new RotatorAscom(choice);
         }
+#endif
+#ifdef ROTATOR_INDI
+        else if (choice.Contains(_T("INDI"))) {
+			rotator = INDIRotatorFactory::MakeINDIRotator();
+		}

```

Comment:
nit: please setup your editor to use spaces and not hard tabs and use 4-space indentation
```suggestion
            rotator = INDIRotatorFactory::MakeINDIRotator();
        }
```